MovieChecker/MovieChecker/RottenTomatoes.h

```diff
@@ -0,0 +1,18 @@
+//
+//  RottenTomatoes.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface RottenTomatoes : NSObject
+
++ (NSURL *)appRootURL;
++ (NSString *)apiKey;
++ (NSURL *)boxOfficeURL;
++ (NSArray *)arrayFromData:(NSData *)data withKey:(NSString *)key;
```

Comment:
Why pass in the key?  Would it make sense for the key to be a constant within this class's implementation?